linera-sdk/src/graphql.rs

```diff
@@ -2,12 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! GraphQL traits for generating interfaces into applications.
-pub use linera_sdk_derive::MutationRoot;
-/// A trait which is derived by the `MutationRoot` proc macro.
+
+/// Expose the derive macro for [`GraphQLMutationRoot`].
```

Comment:
Nit (squared?): I'd suggest using `export` instead of `expose`, because usually the Rust documentation includes a "re-exports" section (e.g.: https://docs.rs/tokio/latest/tokio/#reexports).

```suggestion
/// Export the derive macro for [`GraphQLMutationRoot`].
```

Reply:
ok. It would be `Exports` btw :)